src/composer.json

```diff
@@ -39,7 +39,7 @@
     
     "config": {
         "platform": {
-            "php": "5.3.3"
```

Comment:
I am uncertain for myself. In turn I think the practise of organisations using ten year old distros just because their IT department does not support other version of Linux  should be revised too. And upgrading PHP is more urgent than upgrading other components to latest stable.

Also I remember correctly now, only @bcornec has complained on the mailing list about upping PHP to 5.6 (because we need this for the securing login with OAuth).

I think there is the point to use cent 7 and this is maybe not even in the epel 7? Then, it could conflict with this branch not merged, right?

https://github.com/fossology/fossology/pull/701